packages/flutterfire_ui/lib/src/i10n/lang/hi.dart

```diff
@@ -0,0 +1,263 @@
+  final String isNotAValidEmailErrorText;
+  @override
+  final String userNotFoundErrorText;
+  @override
+  final String emailTakenErrorText;
+  @override
+  final String accessDisabledErrorText;
+  @override
+  final String wrongOrNoPasswordErrorText;
+  @override
+  final String signInText;
+  @override
+  final String registerText;
+  @override
+  final String registerHintText;
+  @override
+  final String signInHintText;
+  @override
+  final String signOutButtonText;
+  @override
+  final String phoneInputLabel;
+  @override
+  final String phoneNumberIsRequiredErrorText;
+  @override
+  final String phoneNumberInvalidErrorText;
+  @override
+  final String profile;
+  @override
+  final String name;
+  @override
+  final String deleteAccount;
+  @override
+  final String passwordIsRequiredErrorText;
+  @override
+  final String confirmPasswordIsRequiredErrorText;
+  @override
+  final String confirmPasswordDoesNotMatchErrorText;
+  @override
+  final String confirmPasswordInputLabel;
+  @override
+  final String forgotPasswordButtonLabel;
+  @override
+  final String forgotPasswordViewTitle;
+  @override
+  final String resetPasswordButtonLabel;
+  @override
+  final String verifyItsYouText;
+  @override
+  final String differentMethodsSignInTitleText;
+  @override
+  final String findProviderForEmailTitleText;
+  @override
+  final String continueText;
+  @override
+  final String countryCode;
+  @override
+  final String codeRequiredErrorText;
+  @override
+  final String invalidCountryCode;
+  @override
+  final String chooseACountry;
+  @override
+  final String enableMoreSignInMethods;
+  @override
+  final String signInMethods;
+  @override
+  final String provideEmail;
+  @override
+  final String goBackButtonLabel;
+  @override
+  final String passwordResetEmailSentText;
+  @override
+  final String forgotPasswordHintText;
+  @override
+  final String emailLinkSignInButtonLabel;
+  @override
+  final String signInWithEmailLinkViewTitleText;
+  @override
+  final String signInWithEmailLinkSentText;
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const EnLocalizations({
+    this.emailInputLabel = 'Email',

```

Comment:
Seems there are a lot of texts yet to be translated to Hindi :) If this is a work in progress, please consider marking this as a draft :) 
(I'm not from Google. Just sharing some thoughts)

Reply:
I had a question regarding that. I could see some other translations use Email as Email. I thought it'd do for Hindi as well. Should I like translate these too?

Reply:
I suggest adding native translations where ever possible. Also, I noticed that you have used English scripts all over the place for translation. Some languages like Bahasa, Spanish, etc use English script by default and this is not the case for Hindi. For example, 
**Phone number zaroori hai** - Wrong
**फ़ोन नंबर आवश्यक है** - Correct

Please note that I'm not a native Hindi speaker, these are some generic comments and I may not be able to review the translation itself

Reply:
Hey @rahulraj64, I have made the changes. Could you please review it again? Thanks!